content/en/docs/setup/production-environment/tools/kops.md

```diff
@@ -232,5 +232,6 @@ including tools for logging, monitoring, network policy, visualization, and cont
 * Learn more about `kOps` [advanced usage](https://kops.sigs.k8s.io/) for tutorials,
   best practices and advanced configuration options.
 * Follow `kOps` community discussions on Slack:
-  [community discussions](https://github.com/kubernetes/kops#other-ways-to-communicate-with-the-contributors).
-* Contribute to `kOps` by addressing or raising an issue [GitHub Issues](https://github.com/kubernetes/kops/issues).
+  [community discussions](https://kubernetes.slack.com/archives/C3QUFP0QM).   
```

Comment:
I see there's a page on kOps documentation that seems to point to more communication channels: https://kops.sigs.k8s.io/contributing/#other-ways-to-communicate-with-the-contributors

Perhaps that would be a better link to add here, instead of the Slack channel only?

Reply:
@stormqueen1990 I agree with you changed the link PTAL!